### DISCUSSION TITLE
We should redesign/modernize the website !

### DISCUSSION THREAD

**Discussion by louchebem06** [d-r-e/rncp#55] [Ideas]
You could use this component library to redesign our super RNCP.
https://akveo.com/ngx-admin/pages/dashboard?theme=cosmic

**Comment by kakesinfo** [comment]
It is a great idea to refactor the website design and make it much 2024 !

**Comment by d-r-e** [comment]
The design of the website was purposely done in black/white/retro but I'm open to suggestions. The easy way would be to go full tailwind, shadows and rounded corners or something like material design, but I'm really tired of that 😅
